chalice/cli/__init__.py

```diff
@@ -52,7 +52,7 @@ def create_new_project_skeleton(project_name, profile=None):
     # type: (str, Optional[str]) -> None
     chalice_dir = os.path.join(project_name, '.chalice')
     os.makedirs(chalice_dir)
-    config = os.path.join(project_name, '.chalice', 'config.json')
+    config = os.path.join(project_name, '.chalice', 'config.yml')
```

Comment:
Since YAML is a superset of JSON you can make this PR backwards compatible by doing a "does file exist" check for both the `.json` and the `.yml` file and then using the one that exists.

Reply:
Great idea.